package.json

```diff
@@ -34,6 +34,7 @@
     "postcss-loader": "^0.8.0",
     "postcss-reporter": "^1.2.1",
     "postcss-simple-vars": "^1.0.0",
+    "prismjs": "git+https://github.com/PrismJS/prism.git#gh-pages",
```

Comment:
Don't know exactly, but I've found this: https://github.com/tomchentw/react-prism/blob/master/package.json#L79

Also this one: https://github.com/PrismJS/prism/issues/966

But I've just tried to use the @latest from npm and it is looking good.